modular_citadel/code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -29,44 +44,57 @@
 	ratvar_modules = list(/obj/item/clockwork/slab/cyborg/security,
 		/obj/item/clockwork/weapon/ratvarian_spear)
 	cyborg_base_icon = "k9"
-	moduleselect_icon = "security"
+	moduleselect_icon = "k9"
```

Comment:
Make sure you point to a different, modular icon file with the `moduleselect_alternate_icon` var

Reply:
tbh, I already have the huds backed up in the citadel refs, and if anything happens to those files I'll fix 'em.

Reply:
DMI conflicts are a pain in the ass to detect due to how they're handled. It's better to modularize new icons and save the trouble rather than having to readd the new iconstates to the .dmi every single time upstream makes one single change.